### PR TITLE
[bitnami/odoo] Add VIB tests

### PR DESCRIPTION
**Pull request by marcosbc**
### Description of the change

The main objective of this PR is to publish our Bitnami Odoo container using VMware Image Builder. In order to do that, several changes are included:

* Increasing the existing test coverage of the asset by adding Goss tests.
* Update verify and publish VIB pipeline's definitions.

### Benefits

* Ensuring higher quality of the container catalog.
* Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

N/A

### CODE REVIEW

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! I left you a comment as a review, please take a look when you have a chance 😁

File: .vib/odoo/goss/odoo.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+command:
+  check-virtualenv:
+    exec: bash -c '. /opt/bitnami/odoo/venv/bin/activate; pip install tool'
+    exit-status: 0
+    stdout:
+      - /Successfully installed.* tool-/
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/odoo/goss/odoo.yaml]: Added, but please beware that we are not currently doing it for the tests of the Python container:

```yaml
  check-pip:
    exec: pip install docutils
    exit-status: 0
```

[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/odoo/goss/odoo.yaml]: Oops, noted! I will change it as well 😁

File: .vib/odoo/goss/odoo.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+command:
+  check-virtualenv:
+    exec: bash -c '. /opt/bitnami/odoo/venv/bin/activate; pip install tool; pip uninstall tool'
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/odoo/goss/odoo.yaml]: Fixed!

**Review by joancafom — APPROVED** [review]

**Pull request merged**